Clarify MCP Audit Log Scope (#8028)
FlowFuse/flowfuse: docs/user/logs.md

<img src="images/teamauditlog.png" width="500" />

### AI Agents and API Activity

For actions performed through FlowFuse Expert or the FlowFuse platform API, both this log and the instance/application-level [Audit Log](#audit-log) show an icon indicating how the action was performed:

- A sparkle icon means the action was performed by FlowFuse Expert on the user's behalf. Hovering over the icon shows the name of the tool that was called.
- A terminal icon means the action was performed through the FlowFuse API using a [Personal Access Token](./user-settings.md#personal-access-tokens).

Actions performed directly through the FlowFuse UI do not show an icon.

> These logs provide visibility into supported FlowFuse Expert MCP tool actions and API actions only. They do not include all MCP activity, such as actions performed through an instance MCP server.

Reviewer: ```suggestion
These logs provide visibility into supported FlowFuse Expert and API actions only.
```

We are not exposing any MCP endpoint yet so I'd say it's better to not mention MCP yet.

> ### MCP and API Activity

Reviewer: ```suggestion
### AI Agents and API Activity
```

FlowFuse is not yet available with MCP integration so I would either start by saying "AI Agents" or "FlowFuse Expert" only and then update the docs for the MCP integration of FlowFuse

Reviewer: ```suggestion
### FlowFuse Expert and API Activity
```

Reviewer: I'd say to choose one of these two @dimitrieh @sumitshinde-84

Author: I think ai agents is good since it also covers other ai agents using mcp tools

> For actions performed through FlowFuse Expert MCP tools or the FlowFuse platform API, both this log and the instance/application-level [Audit Log](#audit-log) show an icon indicating how the action was performed:

Reviewer: ```suggestion
For actions performed using FlowFuse platform, FlowFuse Expert or the FlowFuse API, both this log and the instance/application-level [Audit Log](#audit-log) show an icon indicating how the action was performed:
```

Reviewer: I might be wrong having added FlowFuse platform (i meant the UI) so feel free to correct me but logs are also created for actions performed using the website, aren't they?

Reviewer: ```suggestion
For actions performed through FlowFuse Expert or the FlowFuse platform API, both this log and the instance/application-level [Audit Log](#audit-log) show an icon indicating how the action was performed:
```

Ok i read down below that no icons are present when actions are done using the UI, my bad

Author: I think flowfuse platform does not show the icon

> - A sparkle icon means the action was performed by FlowFuse Expert through an MCP tool on the user's behalf. Hovering over the icon shows the name of the tool that was called.

Reviewer: ```suggestion
- A sparkle icon means the action was performed by FlowFuse Expert on the user's behalf. Hovering over the icon shows the name of the tool that was called.
```

Yes we are using MCP tools but it's confusing to the user due to the custom MCP server we allow so I would not phrase it in this way
